layers/factory.go

```diff
@@ -5,10 +5,22 @@ import (
 	"path/filepath"
 	"strings"
 
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+
 	"github.com/buildpacks/lifecycle/archive"
 	"github.com/buildpacks/lifecycle/log"
 )
 
+const (
+	BuildpackLayerName      = "Layer: '%s', Created by buildpack: %s"
+	ExtensionLayerName      = "Layer: '%s', Created by extension: %s"
+	LauncherConfigLayerName = "Buildpacks Launcher Config"
```

Comment:
Should this be named a little more generic given the layer name?
```suggestion
	LauncherConfigLayerName = "Buildpacks Config"
```

Reply:
I left as-is for now because https://github.com/buildpacks/rfcs/blob/main/text/0102-history.md has "launcher", but we can change it if anyone has a strong opinion